Sources/AWSSignerV4/signer.swift

```diff
@@ -172,38 +173,36 @@ public struct AWSSigner {
             "\(canonicalHeaders)\n\n" +
             "\(signingData.signedHeaders)\n" +
             signingData.hashedPayload
-        return canonicalRequest
+        return Data(canonicalRequest.utf8)
     }
-    
+
     /// Create a SHA256 hash of the Requests body
     static func hashedPayload(_ payload: BodyData?) -> String {
         guard let payload = payload else { return hashedEmptyBody }
-        let hash : [UInt8]?
+        let hash : String?
         switch payload {
         case .string(let string):
-            hash = sha256(string)
+            hash = SHA256.hash(data: Data(string.utf8)).hexDigest()
```

Comment:
Replace `Data(string.utf8)` with `[UInt8](string.utf8)`. This SHOULD be a non copy operation and we lower the dependency on `Data`.